data/clients.xml

```diff
@@ -284,6 +285,12 @@
             <extensions from="8.20 - 8.31" to="10.100"/>
             <data format="10.57" dat="0x44BA" spr="0x585138E5"/>
 		</client>
+		
+		<client name="12.71 - 12.72" otb="12.71" visible="true" data_directory="1271">
+            <otbm version="3"/>
+            <extensions from="8.20 - 8.31" to="12.71 - 12.72"/>
```

Comment:
broken indent

Reply:
it's following the pattern of the versions above